scripts/DEBUGGING.md

```diff
@@ -0,0 +1,73 @@
+# Debugging
```

Comment:
This is great, but I'd suggest moving it to the `docs` folder and referencing it in the `tutorials` sections of https://github.com/hyperledger-labs/business-partner-agent/tree/master/docs

Reply:
sorry, that was part of another PR #368 .
This file and everything under /local-network is related to that PR.